ts/src/provider.ts

```diff
@@ -230,11 +230,12 @@ export class NodeWallet implements Wallet {
   constructor(readonly payer: Keypair) {}
 
   static local(): NodeWallet {
+    const process = require("process");
     const payer = Keypair.fromSecretKey(
       Buffer.from(
         JSON.parse(
           require("fs").readFileSync(
-            require("os").homedir() + "/.config/solana/id.json",
+            process.env.ANCHOR_WALLET,
```

Comment:
I think the better thing to do here is to read the Anchor.toml directly instead of using environment variables, so that this works outside of `anchor test`. To do that we can just recursively look at parent directories until we find the first Anchor.toml.

Reply:
Mmmmh, in what scenario do we want to run tests without going through `anchor test`?

Reply:
Oh, and i think i forgot about a consumer that will make a node wallet through `@project-serum/anchor`.
But in this case he wouldn't have an Anchor.toml